pkgs/servers/dendrite/default.nix

```diff
@@ -13,6 +13,8 @@ buildGoModule rec {
 
   vendorSha256 = "1l1wydvi0yalas79cvhrqg563cvs57hg9rv6qnkw879r6smb2x1n";
 
+  passthru.tests = nixosTests.dendrite;
```

Comment:
```suggestion
  passthru.tests = {
    inherit (nixosTests) dendrite;
  }
```

Reply:
Is there a reason for this? The former pattern is very common in nixpkgs, and we aren't inheriting multiple tests.

Reply:
It's what every other package does.

Reply:
This isn't true, but I'v changed it to what you suggested:

```
pkgs/servers/web-apps/engelsystem/default.nix
40:  passthru.tests = nixosTests.engelsystem;

pkgs/servers/keycloak/default.nix
58:  passthru.tests = nixosTests.keycloak;

pkgs/servers/miniflux/default.nix
34:  passthru.tests = nixosTests.miniflux;

pkgs/servers/pinnwand/steck.nix
25:  passthru.tests = nixosTests.pinnwand;

pkgs/servers/pinnwand/default.nix
46:  passthru.tests = nixosTests.pinnwand;

pkgs/servers/nextcloud/default.nix
16:    passthru.tests = nixosTests.nextcloud;

pkgs/applications/audio/musescore/default.nix
44:  passthru.tests = nixosTests.musescore;

pkgs/applications/networking/irc/convos/default.nix
83:  passthru.tests = nixosTests.convos;

pkgs/applications/misc/keepassx/community.nix
123:  passthru.tests = nixosTests.keepassxc;

pkgs/os-specific/linux/firejail/default.nix
72:  passthru.tests = nixosTests.firejail;

pkgs/os-specific/linux/usbguard/default.nix
77:  passthru.tests = nixosTests.usbguard;

pkgs/tools/security/bitwarden_rs/vault.nix
19:  passthru.tests = nixosTests.bitwarden;

pkgs/tools/security/bitwarden_rs/default.nix
38:  passthru.tests = nixosTests.bitwarden;
```

Reply:
You're right, but most of them do it correctly, i.e. as documented: https://nixos.org/manual/nixpkgs/unstable/#var-meta-tests.